### PR TITLE
Renamed camera to right_camera

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by wkentaro** [comment]
It should be left hand?

**Comment by yuyu2172** [comment]
right

File: jsk_2016_01_baxter_apc/launch/setup_creative.launch [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
       use_passthrough_filter: false
       use_radius_outlier_filter: false
       use_voxel_grid_filter: false
     </rosparam>
-    <param name="camera_link" type="string" value="/softkinetic_camera_link" />
-    <param name="rgb_optical_frame" type="string" value="/softkinetic_camera_rgb_optical_frame" />
-    <param name="depth_optical_frame" type="string" value="/softkinetic_camera_depth_optical_frame" />
+    <param name="camera_link" type="string" value="/right_softkinetic_camera_link" />
+    <param name="rgb_optical_frame" type="string" value="/right_softkinetic_camera_rgb_optical_frame" />
+    <param name="depth_optical_frame" type="string" value="/right_softkinetic_camera_depth_optical_frame" />
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/setup_creative.launch]: right -> left ?

**Comment by yuyu2172** [comment]
ops. sorry...

**Pull request merged**